tests/Behat/Context/FeatureContext.php

```diff
@@ -9,6 +9,25 @@
 }
 class FeatureContext extends SilverStripeContext
 {
+    /**
+     * @Then /^I should( not |\s*)see the edit form for block (\d+)$/i
```

Comment:
Super minor, but I think ` not |\s*)` should be `\s+` otherwise it might match `I shouldsee the` when it should only match `I should see`